.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.1
+  - 1.4
+
+before_script:
+  - FIXED=$(go fmt ./... | wc -l); if [ $FIXED -gt 0 ]; then echo "gofmt - $FIXED file(s) not formatted correctly, please run gofmt to fix this." && exit 1; fi
+
+script:
+  - go test -v ./...
```

Comment:
Right, well - they both should do the same, but I forgot that's not actually true - I use -gocheck.v when running tests in a given subdir and it gives nicer output - it does choke when no *_test.go files exist in the root dir though.

Fair enough - no need to go get gocheck then.